modules/emulator/NAMESPACE

```diff
@@ -4,6 +4,8 @@ S3method(calcSpatialCov,list)
 S3method(calcSpatialCov,matrix)
 S3method(p,jump)
 S3method(p,mvjump)
+S3method(plot,jump)
```

Comment:
That's a question for @mdietze , it's his function. It _looks_ like `jump` is an S4 class, so I don't think that's right. I changed it in cbdf8b0, but I can revert if necessary.